### PR TITLE
Improves xbutil query output

### PR DESCRIPTION
**Pull request by rradjabi**
~~I need to rewrite how values are retrieved from Mem and CU so that I can use the get_pretty function for them.~~
Addressed this in 43a6b8d

### CODE REVIEW

File: src/runtime_src/driver/xclng/tools/xbutil/xbutil.h [hunk 1]
```diff
@@ -337,7 +338,7 @@ class device {
         if(buf.empty())
             return;
 
-        for(unsigned i = 0; i < (unsigned)map->m_count; i++) {
+        for(int32_t i = 0; i < map->m_count; i++) {
```
Comment (rradjabi) [inline src/runtime_src/driver/xclng/tools/xbutil/xbutil.h]: make int

**Comment by rradjabi** [comment]
Ready to merge. Sonal and I did a code review.

**Comment by NaAl-Xlnx** [comment]
compilation error:
18:06:59 [Build XRT and create DEB on Ubuntu16.04] /opt/xrt/src/runtime_src/driver/xclng/tools/xbutil/xbutil.h:853:62: error: 'cu_i' may be used uninitialized in this function [-Werror=maybe-uninitialized]
18:06:59 [Build XRT and create DEB on Ubuntu16.04]                ostr << "CU[" << std::right << std::setw(2) << cu_i << "]: "
18:06:59 [Build XRT and create DEB on Ubuntu16.04]                                                               ^
18:06:59 [Build XRT and create DEB on Ubuntu16.04] /opt/xrt/src/runtime_src/driver/xclng/tools/xbutil/xbutil.h:791:60: error: 'index' may be used uninitialized in this function [-Werror=maybe-uninitialized]
18:06:59 [Build XRT and create DEB on Ubuntu16.04]                     << "[" << std::right << std::setw(2) << index << "] " << std::left
18:06:59 [Build XRT and create DEB on Ubuntu16.04]                                                             ^
18:06:59 [Build XRT and create DEB on Ubuntu 18.04] [ 96%] Building CXX object runtime_src/xocl/CMakeFiles/xocl.dir/core/platform.cpp.o
18:06:59 [Build XRT and create DEB on Ubuntu 18.04] [ 96%] Building CXX object runtime_src/xocl/CMakeFiles/xocl.dir/core/program.cpp.o
18:06:59 [Build XRT and create DEB on Ubuntu 18.04] [ 96%] Building CXX object runtime_src/xocl/CMakeFiles/xocl.dir/core/sampler.cpp.o
18:06:59 [Build XRT and create DEB on Ubuntu 18.04] [ 97%] Building CXX object runtime_src/xocl/CMakeFiles/xocl.dir/core/stream.cpp.o
18:06:59 [Build XRT and create DEB on Ubuntu 18.04] [ 97%] Building CXX object runtime_src/xocl/CMakeFiles/xocl.dir/xclbin/xclbin.cpp.o
18:07:00 [Build XRT and create DEB on Ubuntu 18.04] [ 97%] Built target hw_emu_objects
18:07:00 [Build XRT and create RPM on Centos7.4] [ 54%] Building CXX object runtime_src/xocl/CMakeFiles/xocl.dir/api/clEnqueueCopyBufferRect.cpp.o
18:07:00 [Build XRT and create RPM on Centos7.4] [ 54%] Built target sw_emu_objects
18:07:00 [Build XRT and create RPM on Centos7.4] [ 54%] Building CXX object runtime_src/xdp/CMakeFiles/xdp.dir/profile/profile_device/xdp_profile_writers.cpp.o
18:07:00 [Build XRT and create RPM on Centos7.4] [ 54%] Building CXX object runtime_src/tools/xclbin/CMakeFiles/xclbinutil.dir/SectionKeyValueMetadata.cxx.o
18:07:01 [Build XRT and create RPM on Centos7.4] [ 55%] Building CXX object runtime_src/xdp/CMakeFiles/xdp.dir/profile/profile_device/xperf.cpp.o
18:07:01 [Build XRT and create RPM on Centos7.4] [ 55%] Building CXX object runtime_src/driver/xclng/tools/xbflash/CMakeFiles/xbflash.dir/xspi.cpp.o
18:07:01 [Build XRT and create RPM on Centos7.4] [ 55%] Building CXX object runtime_src/xocl/CMakeFiles/xocl.dir/api/clEnqueueCopyBufferToImage.cpp.o
18:07:01 [Build XRT and create RPM on Centos7.4] [ 55%] Building CXX object runtime_src/xocl/CMakeFiles/xocl.dir/api/clEnqueueCopyImage.cpp.o
18:07:01 [Build XRT and create RPM on Centos7.4] [ 56%] Building CXX object runtime_src/xocl/CMakeFiles/xocl.dir/api/clEnqueueCopyImageToBuffer.cpp.o
18:07:01 [Build XRT and create DEB on Ubuntu16.04] cc1plus: all warnings being treated as errors

**Comment by rradjabi** [comment]
retest this please

**Comment by abalasa** [comment]
retest this please

**Comment by rradjabi** [comment]
Validated results on KCU1500 as well as U200. @maxzhen  or @sonals, please merge.

**Comment by rradjabi** [comment]
Addresses JIRA CR-1017524 .

**Pull request merged**